app/javascript/application_webpack.js

```diff
@@ -15,6 +15,9 @@ window.$ = window.jQuery = $;
 import * as ajax_events from "javascripts/ajax_events";
 window.ajax_events = ajax_events;
 
+import * as fetch_proxy from "javascripts/fetch_proxy";
```

Comment:
You can just do `import "javascripts/fetch_proxy";` here. There isn't anything being exported, so this script is being imported for the side effect only (and should be included with the other such imports in this file)